pkgs/tools/compression/offzip/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchzip
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "offzip";
+  version = "0.4.1";
+
+  src = fetchzip {
+    url = "https://aluigi.altervista.org/mytoolz/offzip.zip";
```

Comment:
I believe this URL needs some `archive.org` trick, given that it does not parameterize the version.